packages/appium/lib/schema/schema.js

```diff
@@ -576,17 +579,19 @@ export class SchemaNameConflictError extends Error {
   code = 'APPIUMERR_SCHEMA_NAME_CONFLICT';
 
   /**
-   * @type {Readonly<{extType: ExtensionType, extName: string}>}
+   * @type {Readonly<{extType: ExtensionType, extName: string, conflictedSchema?: SchemaObject}>}
    */
   data;
 
   /**
    * @param {ExtensionType} extType
    * @param {string} extName
+   * @param {{conflictedSchema?: SchemaObject}} [opts]
    */
-  constructor(extType, extName) {
-    super(`Name for ${extType} schema "${extName}" conflicts with an existing schema`);
-    this.data = {extType, extName};
+  constructor(extType, extName, {conflictedSchema} = {}) {
+    super(`Name for ${extType} schema "${extName}" conflicts with an existing schema.`);
+    this.data = {extType, extName, conflictedSchema};
+    logger.debug(JSON.stringify(this.data, null, 2));
```

Comment:
we want to print something just when the error is created? is this kind of an equivalent to our  old pattern of using `errAndThrow`?

Reply:
I tossed this anyway